docs/docs/codex/fenrir/getting_started/index.html

```diff
@@ -81,7 +81,7 @@
     <div class="col-md-3 side-nav-col">
       <ul class="nav navbar-nav side-nav">
         
-          <li><span class="nav-header"><i class="fa fa-hand-o-right"></i> <span>Introduction</span></span><ul class="nav "><li><a href="/docs/introduction/overview/">Overview</a></li><li><a href="/docs/introduction/glossary/">Glossary</a></li></ul></li>
+          <li><span class="nav-header"><i class="fa fa-hand-o-right"></i> <span>Introduction</span></span><ul class="nav "><li><a href="/docs/introduction/ecosystem/">Ecosystem</a></li><li><a href="/docs/introduction/overview/">Overview</a></li><li><a href="/docs/introduction/comparison/">Comparison to alternatives</a></li><li><a href="/docs/introduction/glossary/">Glossary</a></li></ul></li>
```

Comment:
I see Ecosystem in the navigation, but not in the included files of this PR.  Can you either add the file in the PR or move it elsewhere and re-build the site?

Also - Overview is second.  Is that the order what we want? Usually we put the Overview first in the navigation because it's more of the introductory page.

Reply:
This was a mistake - I had that change in the same branch & didn't realize it impacted the output.